### PR TITLE
[Fix] Refine the input of transform in pssl dataset

### PR DESCRIPTION
**Pull request by juncaipeng**
* Refine the input of transform in pssl dataset
* Use 1001 in padding

### CODE REVIEW

File: configs/pssl/pp_liteseg_stdc1_pssl.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ train_dataset:
       scale_step_size: 0
     - type: RandomPaddingCrop
       crop_size: [256, 256]
-      label_padding_value: 1000
+      label_padding_value: 1001
```
Comment (wuyefeilin) [inline configs/pssl/pp_liteseg_stdc1_pssl.yml:21]: loss的ignore_index的设置需要保持一致，loss里面默认都是255

**Review by wuyefeilin — APPROVED** [review]
LGTM
